Makefile

```
@@ -22,3 +22,16 @@ lint.charts.kong:
 
 lint.shellcheck:
 	shellcheck ./scripts/*
+
+KONG_CHART_GO_TEST_MODULE=charts/kong/tests
+TEST_GOLDEN_TESTCASE_NAME=TestGolden
```

We might leave the room to specify this when calling `make`

```suggestion
KONG_CHART_GO_TEST_MODULE ?= charts/kong/tests
TEST_GOLDEN_TESTCASE_NAME ?= TestGolden
```